src/components/VmDetails/cards/DetailsCard/index.js

```diff
@@ -89,6 +89,7 @@ NotAvailable.propTypes = {
 }
 
 const DEFAULT_BOOT_DEVICES = List(['hd', null])
+const DEFAULT_GMT_TIMEZONE = timezones.find(timezone => timezone.value.startsWith('(GMT) Greenwich')).id
```

Comment:
Should this come from the redux state.config?

Reply:
I'm not sure if there is something like that. And maybe it worth to check this comment: https://github.com/oVirt/ovirt-web-ui/pull/1253#issuecomment-676298477

Reply:
We query it at login time and then push it to the config reducer:

https://github.com/oVirt/ovirt-web-ui/blob/f4648012a59fba41db467b46e81d6325dd002bd9/src/sagas/server-configs.js#L62-L65

https://github.com/oVirt/ovirt-web-ui/blob/f4648012a59fba41db467b46e81d6325dd002bd9/src/reducers/config.js#L117-L125

Are those default not applicable here?

Reply:
No, unfortunately, they are not applicable.
`defaultGeneralTimezone` has the value `Etc/GMT` and
`defaultWindowsTimezone` has the value `GMT Standard Time`.